knowledge-portal/central/initiate-push-to-source.sh

```diff
@@ -2,21 +2,12 @@
 
 set -e
 
-# Check and reset to previous commit until last 5 commits, if message contains "docs:" keyword; else break out of loop 
-commits_to_check=1
-while [ $commits_to_check -le 5 ]
-do
-    commit_message=$(git log -1 --format=%B);
-    if [[ ${commit_message} == *"docs:"* ]]
-    then
-        git reset --soft HEAD~1
-        ((commits_to_check++))
-    else
-        break
-    fi
-done
```

Comment:
Are we removing the system where we reset to the n-th last commit ?

Reply:
Yeah, the pipeline will only take the latest commit.